EventSourcing.Core/Migrations/EventMigratorBase.cs

```diff
@@ -0,0 +1,23 @@
+namespace EventSourcing.Core.Migrations;
+
+
+public abstract class EventMigratorBase<TSourceEvent, TTargetEvent> : IEventMigrator 
```

Comment:
Maybe just call it EventMigrator? Is more in line with e.g. Aggregate, which is also abstract?